src/main/resources/application.yaml

```diff
@@ -355,14 +355,155 @@ darts:
+          at-most-for: PT45M
+      generate-case-document-for-retention-date:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      inbound-transcription-annotation-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
       inbound-audio-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
         unstructured-minimum-duration: 24h
         default-batch-size: 100000
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
       inbound-to-unstructured:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
         threads: 20
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      manual-deletion:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      outbound-audio-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      process-arm-response-files:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      process-arm-rpo-pending:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT5H
+      process-daily-list:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT5H
+      process-dets-to-arm-response:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      remove-duplicated-events:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      unstructured-transcription-annotation-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      unstructured-audio-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        minimum-duration-in-unstructured: 210d #10 weeks
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      unstructured-to-arm:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      arm-rpo-duration: ${ARM_RPO_DURATION:1h}
```

Comment:
Not sure the config around this part is right. It doesn't have the `system-user-email` and `lock` that other tasks use. Then the same for `process-e2e-arm-rpo`. Then you have `thread-pool-size:` and  `system-user-email` which don't look like they are in the right place, please can you review?

Reply:
These are all existing unchanged configuration items.

However, I have moved some of these config items around to standardise there location